### ISSUE TITLE
issue B: npmjs.com shows old repo URL (but navigate to latest)

### ISSUE THREAD

**Issue by alundiak** [npm/www#128]
Initial repo URL was https://www.npmjs.com/package/test
I made first publish of `v1.0.7` and then renamed repo name to
https://www.npmjs.com/package/feby
And when I go now to main page on npmjs.com, I see URL to old repo, but when I click it, it goes to proper repo.

<img width="319" alt="npmjscom_old_repo_url_still_on_view" src="https://cloud.githubusercontent.com/assets/2131633/23321336/557fe942-fadf-11e6-91fc-9d85bb56e9e3.png">
U see, screenshot fone, when it was 1.0.7 published yet.
And even now, published 1.0.8, and it stills shows the same old url.

**Comment by ashleygwilliams** [comment]
:wave: hello!

the details on the package page are populated with data from your `package.json`. i took a look in your repo and found your package.json still lists `github.com/alundiak/test` - https://github.com/alundiak/feby/blob/master/package.json#L11. 

if you update your `package.json`, bump the version, and publish, you'll see the updated changes!

good luck, and thanks for filing :)

**Comment by alundiak** [comment]
@ashleygwilliams thanks. I modified code, published and "not healthy" behavior is fixed.

**Comment by ashleygwilliams** [comment]
awesome! glad it worked. don't hesitate to reach out again with any qs :) we are super happy to help

**Comment by alundiak** [comment]
@ashleygwilliams kind related to this, but not yet sure. 
I had 1.0.8 release of my test module, and afai remember it was with url `/test`.
Then I published new 1.0.9 release, ith `/feby` as u suggested in package.json.
Today, I checked again npm registry, and I've seen 1.0.9 but with bad-old url (`/test`).
Anyway, I think issue is resolved, but I can't reproduce it in the same sequence, to be sure it's fixed.
I mena, I wanted to change package.json back again to bad url, then change and publish. BUT I faced with new bug


I did `unpublish`-ed all versiones I had as tags in repo, I removed all tags from repo. I also removed local tags in working copy. But NPM stuck on 1.0.8 version/release/tag/not-sure. Anyway I can do this:

```
npm unpublish feby@1.0.8
```

It says:

```
npm unpublish feby@1.0.8
npm ERR! code E400
npm ERR! Registry returned 400 for DELETE on http://registry.npmjs.org/feby/-rev/10-9ceb650e1bd93e5006895c29725c411e
```

Note here, that I've done a few times today, when my codebase was on master branch and at least one more fresh commit. I suspect, NPM can't delete/unpublish module, because it's only one version should be in sue - not sure if it's true restriction/requirement from NPM registry. Guess.

Anyway, my note point here, that I did then commit amended and pushed with `--force`.
And now I'm not sure if it's related or not to above warning.

I essentially want to get rid of https://www.npmjs.com/package/feby from web - meaning - I want properly unpublish my test module.


Then, I went further, I modified version to `1.0.10`, but again with amending commit and pushing with force to master.
```
npm publish feby                                         
```

And it says:
```
npm ERR! publish Failed PUT 403
npm ERR! code E403
npm ERR! "You cannot publish over the previously published version 1.0.8." : feby
```

So now, I can't publish new version of my module.

@ashleygwilliams not sure, if here is some new bug, u tell me. But if so, I will create separate issue to maintain bug fix when needed. Meanwhile, I need your help to understand the flow and potential my wrong steps/etc.
